src/NUnitCommon/nunit.agent.core/Runners/TestAgentRunner.cs

```diff
@@ -16,6 +16,8 @@ namespace NUnit.Engine.Runners
     /// </summary>
     public abstract class TestAgentRunner : ITestEngineRunner
     {
+        Logger log = InternalTrace.GetLogger(typeof(TestAgentRunner));
+
```

Comment:
Either remove or if left in for a reason, add "private readonly" keywords and prefix with `_` to match other fields in this class.

Reply:
Agreed on "private readonly" - but it's "log" everywhere else in the app Exception that proves the rule? (What does that mean anyway?) :-)